query07.sql

```diff
@@ -3,4 +3,10 @@
 */
 
 -- Enter your SQL query here
-select ...
+SELECT COUNT(*) AS num_trips_spanning_2days FROM 
+  (SELECT trip_id, duration, date_part('day',start_time::date) AS start_day, date_part('day',end_time::date) AS end_day
+  FROM indego_trips_2019_q2
+  UNION ALL
+  SELECT trip_id, duration, date_part('day',start_time::date) AS start_day, date_part('day',end_time::date) AS end_day 
+  FROM indego_trips_2020_q2) AS u
+WHERE start_day != end_day
```

Comment:
This will actually over-count. Even though the max duration is 1440, some trips do actually start and end more than one day apart.